howtos/v3_to_v4.rst

```diff
@@ -3,14 +3,15 @@
 Migrating from Legacy V3 to V4 "Built on openSUSE"
 ==================================================
 
-Rocksor v3 was based on CentOS 7 and is no longer supported.
+Rockstor v3 was based on CentOS 7 and is no longer supported.
 Last updates per channel:
 
 - Stable (3.9.2-57) - April 2020
 - Testing (3.9.1-16) - November 2017
 
-Our `downloadable <https://rockstor.com/dls.html>`_ v4 installers have Stable channel release
-(`4.1.0 <https://forum.rockstor.com/t/v4-built-on-opensuse-stable-channel-changelog/8212>`_) - Jan 2022 pre-installed.
+Our `downloadable <https://rockstor.com/dls.html>`_ v4 installers have the most Stable channel
```

Comment:
```suggestion
Our `downloadable <https://rockstor.com/dls.html>`_ v4 installers aim to have the most recent Stable channel
```

Reply:
@Hooverdan96 This fixes the missing 'recent' and gives intent: which we currently are falling short on - and likely will from time to time. I.e. my pending task of rebuilding the installers !!

Reply:
yes, that sounds good. I will accept that edit.